dist/logzioBlobAzureFunction/core.js

```diff
@@ -47,30 +53,43 @@ function sendData(data, logType, context) {
   logzioShipper.sendAndClose(callBackFunction);
 }
 
-function getData(url, compressed, callback) {
+function extractFileType(url, isCompressed){
+  const urlArray = url.split(".");
+  if (isCompressed){
+    urlArray.pop();
+  }
+  const fileType  = urlArray.pop();
+  var format = process.env.Format;
+  if ([null, undefined].includes(format)){
```

Comment:
It's required so why do we need this if?

Reply:
It has a default value so it isn't marked as "required" on the template.
Makes sure this is the default even if they erase it..

Reply:
so let's check that format is json, text, or csv and if not use text as default